tests/src/JIT/Performance/CodeQuality/SIMD/SeekUnroll/SeekUnroll.cs

```diff
@@ -143,6 +143,11 @@ public static bool Test(object boxedIndex)
     public static int Main()
     {
         int failures = 0;
+        if (!Vector.IsHardwareAccelerated)
+        {
```

Comment:
1. Might want to add a comment explaining this.
2. Have you verified that this fixes the problem when run through the test harness?  It just now occurred to me that maybe `Main` doesn't execute when xunit-perf runs the test?  (also, I'm not sure if our CI is running this through xunit-perf or not... presumably if we do want to track the performance of this test on a non-accelerated platform we'd want some value between 1 and 100000000...)